tests/test_data/config_file_test.txt

```diff
@@ -1,4 +1,5 @@
 WorkDir=tests/test_data/directories/
+GlobalDataPath=tests/test_data/directories/global/
```

Comment:
Please use `ForcingPath` or `InitialConditionPath` keys to introduce different datasets. If needed, add a new config file with new paths.